lib/honeybadger/config/ruby.rb

```diff
@@ -86,6 +86,18 @@ def backtrace_filter
         get(:backtrace_filter)
       end
 
+      def before_notify(action = nil, &block)
+        hash[:before_notify] ||= []
+
+        if action
+          hash[:before_notify] << action
+        elsif block_given?
+          hash[:before_notify] << block
+        else
+          get(:before_notify)
```

Comment:
This is a nice-to-have, but I wrote it before I realized that this configuration gets saved as a `Hash` on the containing `Honeybadger::Config` object. We could remove it and I don't know that the lost functionality would ever be missed.

Reply:
We should probably remove this to avoid future confusion :slightly_smiling_face: